anchor/common/qbft/src/lib.rs

```
@@ -549,15 +553,23 @@ where
         let mut max_prepared_round = 0;
         let mut max_prepared_msg = None;
 
+        // Deserialize round change justifications for validation
+        let signed_rc_justifications: Vec<SignedSSVMessage> = msg
+            .qbft_message
+            .round_change_justification
+            .iter()
+            .filter_map(|bytes| SignedSSVMessage::from_ssz_bytes(bytes).ok())
+            .collect();
```

If deserialization fails, it will be silently ignored. In conjunction with #577, do we maybe catch this with an `UndecodableNestedMessage` error or something?

(also to be found in other places)

---

this branch doesnt have qbft errors so will adjust in that branch